src/makers/hummingbird/hotspots.ts

```diff
@@ -0,0 +1,41 @@
+import HotspotIcon from './hummingbird.svg'
+import { MakerHotspot } from '../hotspotMakerTypes'
+import ANTENNAS from './antennas'
+
+const HUMMINGBIRD_H500 = {
+  name: 'Hummingbird H500',
+  icon: HotspotIcon,
+  onboardType: 'BLE',
+  translations: {
+    en: {
+      diagnostic:
+        '<b><white>Diagnostic support allows Hummingbird Maker to identify issues with your Hotspot in a secure way.</white></b>\n\nHummingbird will never have access to private keys and will only ever be able to access your Hotspot and not any other devices on your Network.\n\nIf you would like to opt-out of diagnostic support please refer to <purple><b>https://www.xdt.com/#faq</b></purple>for troubleshooting guide or use local diagnostic tools to view real-time operating data of hotspots',
```

Comment:
You're missing a space here
 `<purple><b>https://www.xdt.com/#faq</b></purple>for`
should be 
` <purple><b>https://www.xdt.com/#faq</b></purple> for`